.github/workflows/job_PreCommit_PythonAutoformatter.yml

```diff
@@ -0,0 +1,27 @@
+name: Precommit Python Formatter
+run-name: Python Formatter
+on:
+  pull_request:
+    branches: [ "master", "release-*" ]
+    paths: [ "sdks/python/apache_beam/.*"]
+  issue_comment:
+    types: [created, edited]
+
+
+jobs:
+  PythonFormatter:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, 'Run PythonLint PreCommit')
```

Comment:
Also, I think this is the wrong condition. This supports comment triggers, but it will evaluate to false if github.event.comment.body is not defined (which will be the case for pull_request events

Reply:
Same comment applies to rest of the PR